### PR TITLE
adds v2 content to login modal

### PR DESCRIPTION
**Pull request by CaitHawk**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Related issue(s)

[Jira Ticket]()

## Testing done

- unit testing
- feature flag

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

- The Login Modal
- The Unified Sign In Page

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/platform/user/authentication/components/LoginNote.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-      <strong>September 30, 2025.</strong>
-      <br />
-      <a href="https://www.va.gov/resources/creating-an-account-for-vagov">
-        Learn more about creating a Login.gov or ID.me account
-      </a>
-    </p>
+    {!isV2 && (
+      <p className="vads-u-font-size--base vads-u-margin-top--3 vads-u-padding-x--1">
+        <strong>Note:</strong> We're moving to a simpler and more modern sign-in
+        experience. We'll remove the My HealtheVet option after
```
Comment (asg5704) [inline src/platform/user/authentication/components/LoginNote.js:8]: Ensure there is a space between both `after` words and the date.
Screenshot 2024-09-09 at 3 48 58 PM

File: src/platform/user/authentication/components/LoginNote.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+        <strong>January 31, 2025,</strong> and the DS Logon option after
+        <strong>September 30, 2025.</strong>
+      </p>
+    )}
+    <a
+      href="https://www.va.gov/resources/creating-an-account-for-vagov"
+      className="vads-u-padding-x--1"
```
Comment (asg5704) [inline src/platform/user/authentication/components/LoginNote.js:15]: Add the `target="_blank"` and an `aria-label="Creating a Login.gov or ID.me account. (Opens in a new window)"`

File: src/platform/user/authentication/components/LoginActions.jsx [hunk 3]
```diff
@@ -33,38 +35,79 @@ export default function LoginActions({ externalApplication }) {
...
+        ))}
+        <LoginNote isV2={isSignInV2} />
+        {isSignInV2 &&
+          isValid && (
+            <div>
+              <h2>Other sign-in options</h2>
+              <h3 className="vads-u-margin--0">My HealtheVet sign-in option</h3>
+              <p>
+                <strong>Available through January 31, 2025</strong>
+              </p>
```
Comment (asg5704) [inline src/platform/user/authentication/components/LoginActions.jsx:61]: I believe this should be a part of the `h3`. Probably best to add it as a `<span className="vads-u-display--block">` to get it to wrap appropriately.

File: src/platform/user/authentication/components/LoginActions.jsx [hunk 4]
```diff
@@ -33,38 +35,79 @@ export default function LoginActions({ externalApplication }) {
...
+              <p>
+                You'll still be able to use the <strong>My HealtheVet</strong>{' '}
+                website after this date.
+                <br />
```
Comment (asg5704) [inline src/platform/user/authentication/components/LoginActions.jsx:65]: No `<br />` needed, let it get wrapped by the container.

File: src/platform/user/authentication/components/LoginActions.jsx [hunk 5]
```diff
@@ -33,38 +35,79 @@ export default function LoginActions({ externalApplication }) {
...
+                You'll just need to start signing in with a{' '}
+                <strong>Login.gov</strong> or <strong>ID.me</strong> account.
+              </p>
+              <LoginButton csp="mhv" useOAuth={useOAuth} />
+              <h3 className="vads-u-margin-bottom--0 vads-u-margin-top--3">
+                DS Logon sign-in-option
+              </h3>
+              <p className="vads-u-margin-top--0">
+                <strong>Available through September 25, 2025</strong>
+              </p>
```
Comment (asg5704) [inline src/platform/user/authentication/components/LoginActions.jsx:75]: Same thing. This should be in the `h3` as a `span`.

File: src/platform/user/authentication/components/LoginHeader.jsx [hunk 6]
```diff
@@ -14,7 +16,7 @@ export default function LoginHeader({ loggedOut }) {
             id="signin-signup-modal-title"
             className="vads-u-margin-top--2 medium-screen:vads-u-margin-top--1 medium-screen:vads-u-margin-bottom--2"
           >
-            Sign in
+            {isSignInV2 ? 'Sign in or create an account' : 'Sign in'}
```
Comment (asg5704) [inline src/platform/user/authentication/components/LoginHeader.jsx:19]: 🚀

**Review by asg5704 — CHANGES_REQUESTED** [review]
Only minor changes required.

Also I think the `LoginButton` component will need to be updated.
- [x] Add `ariaDescribedBy` as a prop
- [x] The button needs the `aria-describedby={ariaDescribedBy}`
- [x] Add a unique `id` to the `h3 > span` and pass that to the <LoginButton ariaDescribedBy="mhv" for example. (We can also sync if that's confusing 😅

**Review by asg5704 — APPROVED** [review]

**Review by asg5704 — APPROVED** [review]

**Review by powellkerry — APPROVED** [review]

**Pull request merged**